docs/api_docs/python/compiler/mlir_dialects.rst

```diff
@@ -1,6 +1,24 @@
 MLIR Core Dialects
```

Comment:
This page may overlap with https://iree.dev/reference/mlir-dialects/, https://mlir.llvm.org/docs/Dialects/, and https://mlir.llvm.org/docs/Bindings/Python/. Could link to those pages and explain which to reference when.

Reply:
It actually does, yes. So you can access via our API and technically there is no need to document this here. We could also limit to the most important (input) dialects here.